projects/playground/e2e/integration/playground.spec.ts

```diff
@@ -0,0 +1,158 @@
+const EMAIL = Cypress.env('USER_EMAIL');
+const PASSWORD = Cypress.env('USER_PASSWORD');
+
+if (!EMAIL || !PASSWORD) {
+  throw new Error(
+    'You must provide CYPRESS_USER_EMAIL and CYPRESS_USER_PASSWORD environment variables'
+  );
+}
+
+const loginToAuth0 = () => {
+  cy.get('.auth0-lock-form')
+    .should('have.length.above', 1)
```

Comment:
waits until the lock widget is displayed and has more than 1 children view, otherwise, it would try to find elements when the progress bar is still there.